.prettierignore

```diff
@@ -1,8 +1,9 @@
 .deployment
 .github
 .husky
+!.storybook
 .vscode
-.idea
+.yarn
```

Comment:
how is it related to task?

Reply:
> What else changed
Fixed storybook support for typescript and eslint

I found and fixed an issue when linting didn't work for files in `.storybook` folder because of issue in eslint (there is a comment about that). This change just make file exceptions specified in `prettier` exactly the same as for `eslint`